chrome-enable-autoupdates.py

```diff
@@ -86,10 +86,16 @@ def chrome_product_id():
 
 def keystone_registration_framework_path():
     """Returns KeystoneRegistration.framework path"""
-    keystone_registration = os.path.join(chrome_path, 'Contents/Versions')
-    keystone_registration = os.path.join(keystone_registration, chrome_version())
-    keystone_registration = os.path.join(keystone_registration, 'Google Chrome Framework.framework')
-    keystone_registration = os.path.join(keystone_registration, 'Frameworks/KeystoneRegistration.framework')
+    if int(chrome_version().split(".", 1)[0]) >= int(75):
```

Comment:
I think this would be better with `if LooseVersion(chrome_version()) >= LooseVersion("75"):` and of course adding `from distutils.version import LooseVersion` to imports.

Reply:
Made the requested changes in [8b4c0e5](https://github.com/hjuutilainen/adminscripts/pull/13/commits/8b4c0e5348c7e03d9f86ba6cee02eb83511eb5d5).